### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by Ryooshuu**

### CODE REVIEW

File: Kumi.Game/KumiGameBase.cs [hunk 1]
```diff
@@ -87,13 +81,17 @@ private void load()
             RelativeSizeAxes = Axes.Both,
             Child = new DrawSizePreservingFillContainer() // TODO: Add a way to change the resolution and UI scale dynamically.
             {
-                TargetDrawSize = new Vector2(1920, 1080),
+                TargetDrawSize = new Vector2(1280, 720),
```
Comment (furiner) [inline Kumi.Game/KumiGameBase.cs:84]: i don't believe using a resolution lower than FHD is suitable for most modern displays anymore; especially since there's a lack of ui scaling

before moving the base back to 720p, i'd reccomend actually implementing that feature beforehand

File: Kumi.Game/Screens/Select/ChartSetInfoWedge.cs [hunk 2]
```diff
@@ -15,10 +15,10 @@ public partial class ChartSetInfoWedge : Container
 
     public ChartSetInfoWedge()
     {
-        X = -5;
+        X = 5;
         Masking = true;
         CornerRadius = 5;
-        Margin = new MarginPadding { Left = -5 };
```
Comment (furiner) [inline Kumi.Game/Screens/Select/ChartSetInfoWedge.cs:21]: i get the difference, but why the change? is it really a fix if it's kinda just done out of thin air?

**Review by furiner — CHANGES_REQUESTED** [review]

File: Kumi.Game/KumiGameBase.cs (hunk 1 above)
Comment (Ryooshuu) [reply Kumi.Game/KumiGameBase.cs:84]: This is for consistency from the constructor.

https://github.com/kikuyodev/kumi/blob/00b5650e801455b8f4ff590897f06d64ef840e66/Kumi.Game/KumiGameBase.cs#L45-L48

Before this PR, nothing here would even work, so you wouldn't have a global keybind container. If we were to go with 1920x1080 as the base size as we previously did before, everything would look a whole lot smaller, which you complained about in the past.

File: Kumi.Game/Screens/Select/ChartSetInfoWedge.cs (hunk 2 above)
Comment (Ryooshuu) [reply Kumi.Game/Screens/Select/ChartSetInfoWedge.cs:21]: This is to move the wedge to the right side, hiding the rounded corners on the right. However, this specific line of code didn't do much, so I removed it. In this PR, the wedge is moved over correctly and matches the same width as it should.

File: Kumi.Game/KumiGameBase.cs (hunk 1 above)
Comment (furiner) [reply Kumi.Game/KumiGameBase.cs:84]: epic

**Review by furiner — APPROVED** [review]
